flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerTest.java

```diff
@@ -87,18 +87,6 @@ public void testUnrecognizableAwsRegionInConfig() {
 	}
 
 	@Test
-	public void testCredentialProviderTypeDefaultToBasicButNoCredentialsSetInConfig() {
```

Comment:
We should actually add a test to replace this, for the new behaviours. @tony810430 would you like to add it?